docs/pages/variables.md

```diff
@@ -356,51 +356,32 @@ variation_groups:
           @size-code:           13px; // Custom size only for Mono code blocks
 
           ```
-      - variation_name: Fonts
+      - variation_name: Font stack
         variation_description: >-
-          #### Avenir
+          Override this variable in order to specify a font stack
+          other than the CFPB's default brand font and fallback.
 
           ```
-
-          @webfont-regular:     'AvenirNextLTW01-Regular';
-
-          @webfont-italic:      @webfont-regular;
-
-          @webfont-medium:      'AvenirNextLTW01-Medium';
-
-          @webfont-demi:        @webfont-medium;
-
+          @font-stack: 'Avenir Next', Arial, sans-serif;
           ```
-
-          *Note: We don’t serve the font file for Avenir Next Italic because we found Avenir Next Regular with browser-created faux italics was an acceptable substitute, and it saves a lot of bytes not to serve it.*
-
-
-          *Similarly, we’re trying out Avenir Next Medium with faux bolding in place of Avenir Next Demi, though the results of that experiment have been less predictable, so we may yet revert that decision.*
-
-
-          #### Arial
+      - variation_name: @font-face file path
```

Comment:
@Scotchester Looks like the YAML doesn't like the `@` here.

Reply:
Gah 🤦‍♂️ Thanks, should be fixed now.